src/descrambler/tvhcsa.c

```diff
@@ -220,12 +220,16 @@ tvhcsa_set_type( tvhcsa_t *csa, struct mpegts_service *s, int type )
 }
 
 
-void tvhcsa_set_key_even( tvhcsa_t *csa, const uint8_t *even )
+void tvhcsa_set_key_even( tvhcsa_t *csa, const uint8_t *even, const uint8_t ecm)
```

Comment:
You accidentally removed the space here at the end of the arguments.

Also please squash your commits together.